python/py-requests-cache/Portfile

```diff
@@ -18,29 +17,43 @@ maintainers         nomaintainer
 description         Transparent persistent cache for py-requests
 long_description    ${description}
 
-checksums           rmd160  511d993d81abeffca4d13414a036be94c5c9ce23 \
-                    sha256  1ea8b866db2b16a9b058626c7e1bde0cf669e96abc436cdef352f60237e23ab9 \
-                    size    29527
+homepage            https://requests-cache.readthedocs.io/en/stable/
 
-python.versions     27 35 36 37 38
+checksums           rmd160  41cd04c6a75cd6d10ce08c5e3562ab4517fbe267 \
+                    sha256  27d3eb276ab3affa9864dfc0475241d6d960dd566d57ec46ffa7759c2c74ed1c \
+                    size    1510446
+
+python.versions     27 35 36 37 38 39 410
 
 if {${name} ne ${subport}} {
     depends_build-append \
                     port:py${python.version}-setuptools
 
     depends_lib-append \
-                    port:py${python.version}-requests
+                    port:py${python.version}-appdirs       \
+                    port:py${python.version}-attrs         \
+                    port:py${python.version}-cattrs        \
+                    port:py${python.version}-requests      \
+                    port:py${python.version}-url-normalize \
+                    port:py${python.version}-urllib3
 
     post-destroot {
         set docdir ${prefix}/share/doc/${subport}
         xinstall -d ${destroot}${docdir}
-        xinstall -m 0644 -W ${worksrcpath} README.rst CONTRIBUTORS.rst \
-            LICENSE HISTORY.rst ${destroot}${docdir}
+        xinstall -m 0644 -W ${worksrcpath} README.md CONTRIBUTORS.md \
+            LICENSE HISTORY.md ${destroot}${docdir}
```

Comment:
since these filenames are different in the two version you need to make the distinction here as well with an `if` - `else` block. Something like (untested):
```
if ${python.version} < 36} {
    xinstall -m 0644 -W ${worksrcpath} README.rst CONTRIBUTORS.rst \
            LICENSE HISTORY.rst ${destroot}${docdir}
} else {
    xinstall -m 0644 -W ${worksrcpath} README.md CONTRIBUTORS.md \
            LICENSE HISTORY.md ${destroot}${docdir}
}
```

Reply:
Shoot -sorry, I missed this suggestion (I just noticed github truncated it).  Will fix / clean up with a new update

Reply:
just pushed